src/coreclr/src/vm/excep.h

```diff
@@ -217,12 +217,18 @@ enum UnhandledExceptionLocation
                             FatalExecutionEngineException
 };
 
+#ifdef HOST_WINDOWS
+void InitializeCrashDump();
+bool GenerateCrashDump(LPCWSTR dumpName, int dumpType, bool diag);
+#endif
+
+void CrashDumpAndTerminateProcess(UINT exitCode);
```

Comment:
Yes, it is called on Linux and will generate a crash dump too because the createdump logic is hooked into TerminateProcess in the PAL. I'll add a comment to make that clear.